### DISCUSSION TITLE
Unit 7, Quiz 7.2 part 3

### DISCUSSION THREAD

**Discussion by csnatarajan** [Lightning-AI/dl-fundamentals#77] [Lecture videos and quizzes] [answered]
Shouldn't the number of bias units be equal to the number of output channels? Answer says 3, shouldn't it be 5?

![Screenshot from 2023-07-18 15-43-42](https://github.com/Lightning-AI/dl-fundamentals/assets/10468907/f23936c7-a68f-469b-9573-848be01b0b4a)

**Comment by gracecluvohio** [comment]
That is what I was thinking too.

**Comment by rasbt** [comment, marked as answer]
Sorry, just seeing this. And yes, this is weird. Not sure why it's wrong there!

Anyways, I just updated it! Thanks a lot for reporting that, and sorry for causing this confusion.
